<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registra</class>
 <widget class="QWidget" name="Registra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registrar usuario</string>
  </property>
  <widget class="QLabel" name="usuario">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuario</string>
   </property>
  </widget>
  <widget class="QLabel" name="contrasea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Contraseña</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nuevo_usuario_in">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>291</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nueva_contrasea_in">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>291</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_ingresar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingresar</string>
   </property>
  </widget>
  <widget class="QLabel" name="contrasea_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Reingresar
contraseña</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nueva_contrasea2_in">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>291</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
